packages/ui/package.json

```diff
@@ -48,7 +48,8 @@
 	"dependencies": {
 		"ariakit": "^2.0.0-next.44",
 		"react": "^18.2.0",
-		"react-dom": "^18.2.0"
+		"react-dom": "^18.2.0",
+		"react-icons": "^4.8.0"
```

Comment:
@iCrawl react-icons or @react-icons/all-files?

Reply:
Oh def from all-files

Reply:
So we should use [this](https://github.com/react-icons/react-icons#installation-for-meteorjs-gatsbyjs-etc) instead?

Reply:
Yeah. We already use it elsewhere:

https://github.com/discordjs/discord.js/blob/77191a2e7bb98399766237a1f39eba9d06ab9de6/apps/website/package.json#L55